AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -214,10 +214,6 @@ def test_NodeCategory_ExpandOnClick(self, request, editor, launcher_platform):
 
     def test_VariableManager_UnpinVariableType_Works(self, request, editor, launcher_platform):
         expected_lines = [
-            "Success: VariableManager is opened successfully",
-            "Success: Variable is pinned",
-            "Success: Variable is unpinned",
-            "Success: Variable is unpinned after reopening create variable menu",
         ]
```

Comment:
Do we still need expected lines if it's empty?

Reply:
I just checked with that test runner function and it requires an expected lines argument. I know there are plans to migrate away from Hydra stuff in the future so this will eventually be cleaned up further.